public/external-text/default.yaml

```diff
@@ -0,0 +1,1450 @@
+          your own risk. In no event will the Pacific Climate Impacts Consortium be liable for any loss or damage
+          whatsoever, including, without limitation, indirect or consequential loss or damage, arising from reliance
+          upon the data or derived information.
+
+  credits:
+    title: '# Credits and Acknowledgements'
+    sponsors:
+      title: '## Sponsors'
+      items:
+        - header: Ministry of Transportation and Infrastructure (MoTI)
+          href: https://www2.gov.bc.ca/gov/content/governments/organizational-structure/ministries-organizations/ministries/transportation-and-infrastructure
+          body: Primary sponsor of the PCIC Climate Explorer (PCEX) project.
+    others:
+      title: '## Others'
+      items:
+        - header: Vancouver Island Marmot Recovery Foundation
+          href: https://marmots.org/
+          body: Use of MRF marmot graphic by kind permission.
+    data:
+      title: '## Data'
+      items:
+        - header: Environment and Climate Change Canada
+          href: http://www.ec.gc.ca/
+          body: |
+            We thank the Landscape Analysis and Applications section of the
+            Canadian Forest Service, Natural Resources Canada, for developing
+            and making available the Canada-wide historical daily gridded
+            climate dataset used as the downscaling target.
+
+            PCIC gratefully acknowledges support from
+            Environment and Climate Change Canada
+            for the development of the statistically downscaled GCM
+            scenarios on which much of the data presented here is based.
+
+        - header: World Climate Research Programme
+          href: https://www.wcrp-climate.org/
+          body: |
+            We acknowledge the World Climate Research Programme’s
+            Working Group on Coupled Modelling, which is responsible for
+            CMIP5, and we thank the climate modeling groups for producing
+            and making available their GCM output.
+
+        - header: U.S. Department of Energy
+          href: https://www.energy.gov/
+          body: |
+            For CMIP, the U.S. Department of Energy’s Program for
+            Climate Model Diagnosis and Intercomparison provides coordinating
+            support and led development of software infrastructure in
+            partnership with the Global Organization for
+            Earth System Science Portals.
+
+  contact:
+    title: '# Contact'
+    items:
+      - header: Feedback on Application
+        body: |
+          Please address questions and suggestions on the functioning of this
+          tool (the application proper) to
+          [climate@uvic.ca](mailto:climate@uvic.ca).
+      - header: Scientific Questions
+        body: |
+          Please address questions about science and interpretation of the
+          data presented in this tool to [TBD](#).
```

Comment:
Can this easily be un-linkified? Clicking on it is possible, and takes you back to the data. Not really a serious issue - who's gonna click on it? - but odd.